structure/extension/_EVID.yaml

```diff
@@ -0,0 +1,21 @@
+%YAML 1.2
+---
+contact: https://github.com/glamberson/gedcom-evidence
+extension tags: _EVID
+label: Evidence Reference
+lang: en-US
+payload: XREF
```

Comment:
The value of the payload for a pointer should be the URI it points to, like @<https://github.com/glamberson/gedcom-evidence/_RACT>@ (but replacing that with the correct URI, I'm not sure what it's pointing to.